src/shared/capture/capturev4l.cpp

```diff
@@ -939,23 +929,13 @@ void CaptureV4L::readParameterProperty(VarList * item) {
 #endif
         return;
     }
-    VarDouble * vabs=0;
     VarInt * vint=0;
-    VarBool * vuseabs=0;
     VarBool * venabled=0;
-    VarBool * vauto=0;
-    VarInt * vint2=0;
-    VarInt * vint3=0;
-    VarTrigger * vtrigger=0;
     
     vector<VarType *> children=item->getChildren();
     for (unsigned int i=0;i<children.size();i++) {
         if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="enabled") venabled=(VarBool *)children[i];
-        if (children[i]->getType()==VARTYPE_ID_BOOL && children[i]->getName()=="auto") vauto=(VarBool *)children[i];
```

Comment:
Those variables are once again never read. I've just noticed that these changes should be accompanied by a cleanup of the var type tree.